website/templates/profile/personal_tokens_detail.mako

```diff
@@ -65,9 +65,9 @@
                         <div class="bg-danger f-w-xl token-warning">This is the only time your token will be displayed.</div>
                         <label class="f-w-xl">Token ID</label>
                         <i class="fa fa-info-circle text-muted" data-bind="tooltip: {title:'ID used to authenticate with this token. This will be shown only once.',        placement: 'bottom'}"></i>
-                        <span data-bind="text: token_id" id="token-id-text"></span>
+                        <span data-bind="text: token_id"></span>
```

Comment:
Maybe use a `pre` tag instead of a span tag in order to make the token stand out more and make it more easily copy-able.